packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -70,10 +70,16 @@ Group:      Development/Languages
 
 Conflicts: pulp-consumer-client
 
+%if %{dnf_install}
+Requires: gofer >= 2.12
+%else
 %if 0%{?rhel} == 5
 Requires: gofer >= 2.5
+Requires: gofer < 2.11
 %else
-Requires: gofer >= 2.7.6
+Requires: gofer >= 2.11
+Requires: gofer < 2.12
```

Comment:
Does it actually not work with `2.12`?

Reply:
2.12 requires python 2.7 and above so it will work on el7+